red/nodes/Node.js

```diff
@@ -198,12 +198,22 @@ Node.prototype.error = function(msg) {
 };
 
 Node.prototype.metric = function(msg, eventname, metrics) {
-    metrics = metrics || {};
+    if(metrics && metrics !== null) {
```

Comment:
Yes we assumed that the "metrics" would be an object but without any doc or check to ensure that it was.